docs/content/node-operation/spork.mdx

```diff
@@ -16,28 +16,159 @@ We also utilize a Spork as an opportunity to update the nodes with the latest re
+- **Root State Commit:**0011fda57f2f3aaa8e6bcc1e1deea9778a9543252f8b65bbd4ebca3687789420
+- **Git Commit**: f4a73c7f20109209e9e2e999cf50fcf1ec41241b
+- **Branch/Tag**: v0.13.1
+- **Docker Tag**: v0.13.1
+
+## [Mainnet 3](https://github.com/onflow/flow-go/releases/tag/v0.12.6)
+
+- **Access Node**: access-001.mainnet3.nodes.onflow.org:9000
+- **Date**: Dec 9, 2020
+- **Root Height**: 9737133
+- **Root Parent ID**: 116751c904a7f868cd6e8c90522fdbd70fe826db6886b830338c68c6339df3e7
+- **Root State Commit:**1d2c91e801d0560024848a0c981e03120efc372436ada5f7909c4d44d4600f04
+- **Git Commit**: badd5887512b955e7aa18b4e73dae980ca72fa22
+- **Branch/Tag**: v0.12.6
+- **Docker Tag**: v0.12.6
+
+## Mainnet 2
+
+- **Access Node**: access-001.mainnet2.nodes.onflow.org:9000
+- **Date**: Nov 13, 2020
+- **Root Height**: 8742959
+- **Root Parent ID**: 116751c904a7f868cd6e8c90522fdbd70fe826db6886b830338c68c6339df3e7
+- **Root State Commit:**d6a25be552ed93213df0ffc2e8c7f39f6401c04cbf22bac7a4b84d3c9493f005
+- **Git Commit**: 4ef68efb935c0e3393ae3966752ece5e7739bab4
+- **Branch/Tag**: v0.11.1
+- **Docker Tag**: v0.11.1
+
+## Mainnet 1
+
+- **Access Node**: access-001.mainnet1.nodes.onflow.org:9000
+- **Date**: Oct 13, 2020
+- **Root Height**: 7601063
+- **Root Parent ID**: ab1ee18b6e1c0ee11cc021c26a17c694c627699a576e85f7013cd743bdbc7877
+- **Root State Commit:**6e1adf15689eaf5ea6859bcdd0b510f5eb4c34dac878d8577b3f65bc20c3f312
+- **Git Commit**: 114d45436e7d9052e910c98a1e40f730e3fd12d7
+- **Branch/Tag**: v0.11.1
+- **Docker Tag**: v0.11.1
+
+## Candidate 9
+
+- **Access Node**:
+- **Date**: Sep 25, 2020
+- **Root Height**: 6483246
+- **Root Parent ID**: 9131733835702b0d6321088bddb4642a4964bb5c630440ccb0de47bdbc371d1a
+- **Root State Commit:**90c6f406f5d21880d525ad4702cb249509b85e7f745db2de67e9fe541a56da4c
+- **Git Commit**: v0.9.3
+- **Branch/Tag**: v0.9.3
+- **Docker Tag**:
+
+## Candidate 8
+
+- **Access Node**:
+- **Date**: Sep 25, 2020
+- **Root Height**: 4972987
+- **Root Parent ID**: 5bc2b0900a5138e39d9209a8fe32e14b3e5c884bd36d2a645620f746b7c8bd47
+- **Root State Commit:**6b9161a225b087a461ec95e710fdf4e73f6d6c9401ebf066207a021dced4ce5e
+- **Git Commit**:
+- **Branch/Tag**: v0.8.2
+- **Docker Tag**:
+
+## Candidate 7
+
+- **Access Node**:
+- **Date**: Sep 25, 2020
+- **Root Height**: 4132133
+- **Root Parent ID**: 28f4f495aad016b519acf27fc9d9a328f6a4009807480e36e2df780eeccd99bc
+- **Root State Commit:**001d173bfcf9c7f71684da89bff72b3ee582b39a69c7929360230faf73735c17
+- **Git Commit**: f811f8cd49369ae2bc559e0fbb781aff129484f5
+- **Branch/Tag**: candidate7
+- **Docker Tag**: v0.7.2
+
+## Candidate 6
+
+- **Access Node**:
+- **Date**: Sep 25, 2020
+- **Root Height**: 3187931
+- **Root Parent ID**: 2ff5f7424a448943a153001d2f0869d4fac330906ecb8e17b7ef7fe50e4c7b36
+- **Root State Commit:**bd7f16dc5ef5eced849ab5f437547c14c1907059e1ecf89a942d0521166c5cbb
+- **Git Commit**: b30c48008c0ec1cc8ecb750aeb9ff9f3d712681d
+- **Branch/Tag**: v0.9.3
```

Comment:
this branch/tag doesn't look right

Reply:
I copied and pasted from the table ... could be user error. What is the correct value? Unsure about how to find out.